deno.jsonc

```diff
@@ -19,7 +19,7 @@
     "packages/graphql-server",
     "packages/hello",
     // "packages/medley-router",
-    "packages/node-ws",
+    // "packages/node-ws",
```

Comment:
This one was a bit more complicated..

> TS2741 [ERROR]: Property '#private' is missing in type '{ binaryType: "arraybuffer"; close(code: number | undefined, reason: string | undefined): void; protocol: string; raw: WebSocket; readonly readyState: 0 | 2 | 1 | 3; send(source: string | ... 1 more ... | Uint8Array<...>, opts: SendOptions | undefined): void; url: URL; }' but required in type 'WSContext<WebSocket>'.
        const ctx: WSContext<WebSocket> = {
              ~~~
    at file:///Users/jonno/src/honojs/middleware/packages/node-ws/src/index.ts:117:15
>
>    '#private' is declared here.
      #private!: unknown;
      ~~~~~~~~
        at https://jsr.io/@hono/hono/4.8.3/src/helper/websocket/index.ts:65:35
>
> error: Type checking failed.